esp/services/esdl_svc_engine/esdl_binding.cpp

```
@@ -139,15 +139,21 @@ IPropertyTree * fetchESDLBindingFromDali(const char *process, const char *bindin
 
 IPropertyTree * fetchESDLBindingFromStateFile(const char *process, const char *bindingName, const char * stateFileName)
 {
-    Owned<IPTree> esdlState = createPTreeFromXMLFile(stateFileName);
-    if (!esdlState)
-        throw MakeStringException(-1, "Could not load DESDL binding state from local file: '%s' for ESP binding %s.%s", stateFileName, process, bindingName);
+    try
+    {
+        Owned<IPTree> esdlState = createPTreeFromXMLFile(stateFileName);
+        if (!esdlState)
+            throw MakeStringException(-1, "Could not load DESDL binding state from local file: '%s' for ESP binding %s.%s", stateFileName, process, bindingName);
```

If you have the line 154, why do you need to throw the exception here?